test/src/main/java/org/apache/accumulo/test/functional/ScannerContextIT.java

```diff
@@ -71,7 +72,7 @@ protected int defaultTimeoutSeconds() {
   @Before
   public void checkCluster() throws Exception {
     assumeTrue(getClusterType() == ClusterType.MINI);
-    MiniAccumuloClusterImpl.class.cast(getCluster());
+    assertNotNull(MiniAccumuloClusterImpl.class.cast(getCluster()));
```

Comment:
Would this work better, without warnings? It seems like it would make more sense, if it works.
```suggestion
    assertTrue(MiniAccumuloClusterImpl.class.isInstance(getCluster()));
```

Reply:
Updated and verified that the isInstance worked to remove EP warning.